src/Integration.Vsix/Analysis/DisableRuleCommand.cs

```diff
@@ -95,10 +94,12 @@ private void QueryStatus(object sender, EventArgs args)
         {
             try
             {
-                // Show the menu item for Sonar errors...
-                bool isVisible = TryGetErrorCodeSync(errorList, out _);
```

Comment:
Previously the behaviour was:
* we didn't supported disabling rules in all Sonar languages,
* but if disabling was supported it only worked in standalone mode.

Now the behaviour is slightly more complicated:
* we still don't support disabling rules in all Sonar languages;
* JS rules can always be disabled, regardless of mode,
* other supported languages (i.e. C, C++) can only be disabled in standalone mode.


`TryGetErrorCodeSync`

Reply:
Well, we could simplify the JS stuff, and combine it with C++. As in, if it's not standalone, return false.

Reply:
Yep, we could. That would mean you could disable JS rules in standalone mode, but you'd always have the default settings in connected mode i.e. if a user is already using connected mode and the JS rules are noisy, there's nothing they can do to stop it apart from stop using connected mode or uninstall SLVS.

Reply:
Ah I see, ok 👍